### PR TITLE
13950 Filings UI: broken formatting for draft Change of Company Information

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:* /bcgov/entity#13950

*Description of changes:*
- Fixed css style for `.todo-subtitle`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/TodoList.vue [hunk 1]
```diff
@@ -1740,8 +1740,8 @@ export default class TodoList extends Vue {
 .todo-subtitle {
   color: $gray7;
   display: flex;
-  align-items: center;
   justify-content: flex-start;
+  flex-direction: column;
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:1744]: Make sure this doesn't break any of the other item types.

Comment (lewischenstudio) [reply src/components/Dashboard/TodoList.vue:1744]: I checked this style using other filings, and they were looking fine.

Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:1744]: I think this breaks some specific cases, like this:

image

Check the code for status pending, status error, and status paid.

Comment (lewischenstudio) [reply src/components/Dashboard/TodoList.vue:1749]: In order not to have something like this, I used a different class `.todo-subtitle-warning` for the warning not in good standing instead. The class `.todo-subtitle` remains unchanged.

Comment (severinbeauvais) [reply src/components/Dashboard/TodoList.vue:1749]: Please put the 2 CSS selectors on separate lines and add a space between the key and val on line 1748.

Did you test with most scenarios?

Comment (lewischenstudio) [reply src/components/Dashboard/TodoList.vue:1750]: Yes, I tested this with most scenarios.

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**